src/constants/apiPaths.ts

```diff
@@ -4,7 +4,7 @@ const API_PATHS = {
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://qywkt2ds6g.execute-api.eu-west-1.amazonaws.com/dev',
   bff: 'https://5qpce4wdik.execute-api.eu-west-1.amazonaws.com/dev',
-  cart: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+  cart: 'http://skorykv-cart-api-develop.eu-west-1.elasticbeanstalk.com/api',
```

Comment:
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-http.html
not optimal as well, but maybe less evil that downgrading to http